themes/powerline/README.md

```diff
@@ -5,7 +5,7 @@ A colorful theme, where shows a lot information about your shell session.
 **IMPORTANT:** This theme requires that [a font with the Powerline symbols](https://github.com/powerline/fonts) needs to be used in your terminal emulator, otherwise the prompt won't be displayed correctly, i.e. some of the additional icons and characters will be missing. Please follow your operating system's instructions to install one of the fonts from the above link and select it in your terminal emulator.
 
 **NOTICE:** The default behavior of this theme assumes that you have sudo privileges on your workstation. If it's not the case (i.e you are running on
-a corporate network) you can set the flag 'CAN_SUDO=false' in powerline.theme.bash
+a corporate network) you can set the flag 'THEME_CHECK_SUDO=${THEME_CHECK_SUDO:=false}' in powerline.theme.bash
```

Comment:
When overriding the default behavior, users can simply use `THEME_CHECK_SUDO=false`. Please change this in the documentation.

Also, users shouldn't have to change the `powerline.theme.bash` file when overriding this value. As with all of the other settings, they should simply change it in their `~/.bash_profile` or `~/.bashrc` file. Please change the documentation to reflect this.